apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/ListOrders/CompletedOrders.tsx

```diff
@@ -297,6 +320,33 @@ const TxLink = ({onTxPress, txId}: {onTxPress: () => void; txId: string}) => {
   )
 }
 
+const ListEmptyComponent = ({completedOrders}: {completedOrders: Array<MappedRawOrder>}) => {
+  const {search: assetSearchTerm, visible: isSearching} = useSearch()
+
+  if (isSearching && assetSearchTerm.length > 0 && completedOrders.length === 0) return <EmptySearchResult />
+
+  return <NoOrdersYet />
+}
+
+const NoOrdersYet = () => {
+  const strings = useStrings()
+  return (
+    <View style={styles.imageContainer}>
+      <Spacer height={80} />
+
+      <EmptyCompletedOrdersLogo style={styles.image} />
+
+      <Spacer height={25} />
+
+      <Text style={styles.contentText}>{strings.emptyCompletedOrders}</Text>
+    </View>
+  )
+}
+
+const EmptySearchResult = () => {
+  return null
```

Comment:
I will ask Product for that state since that state is not defined

Reply:
I'm not against, mainly because if later Design decides to add a message for empty search result, this would be exactly what I'll be looking for in this component.